ROS_API.md

```diff
@@ -0,0 +1,29 @@
+Use `rosbot_bringup.launch.py` from `rosbot_bringup` to start all base functionalities for ROSbot 2, 2 PRO, 2R. It consists of following parts:
```

Comment:
Maybe we can use just `bringup.launch.py` name? It will be more consistent with rosbot_xl

Reply:
We will have to change the launch file in every demo in the composes:
https://github.com/husarion/rosbot-mapping/blob/master/compose.rosbot.yaml
https://github.com/husarion/rosbot-navigation/blob/main/compose.rosbot.yaml

Reply:
I haven't noticed that you changed this name :D please update this readme, and yes it will be necessary to update demos, but as we have tagged images used there, first we should build and test new images, and then update demos.

Reply:
Ohh yeah. I missed this too.